tox.ini

```diff
@@ -14,7 +14,7 @@ deps =
 commands =
```

Comment:
Please update from latest master - we just changed out tox configuration. Config has been moved to `pyproject.toml`